test/runtests.jl

```diff
@@ -90,6 +90,18 @@ module wrapper11
     @from "basic.jl" using A, C
 end
 
+module wrapper12
```

Comment:
This module doesn't look like it's been added to the list of modules to test.

Other than, this PR LGTM! Thanks for contributing.